ui/src/Components/Dashboard/Table/TableHeader/SortCursor.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { TABLE } from "../../../../Constant/constant";
+
+export default function SortCursor(props) {
+  const [currentClickedCursor, setCurrentClickedCursor] = useState("");
+
+  const onCursorClick = (clickedCursorType,headerName) =>{
+    setCurrentClickedCursor(clickedCursorType);
+    props.handleSorting(clickedCursorType, headerName);
+  }
+
+  return (
+    <div>
+      <div
+        className={`up-arrow ${
+          currentClickedCursor === TABLE.ASCENDING ? "clicked-u" : ''
```

Comment:
"clicked-u" should be part of the constant file.

Reply:
@Neha clicked-u is a css class name. Just reconfirming if we want this in a constant